src/components/Greeting.js

```diff
@@ -0,0 +1,22 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+/* eslint-disable */
```

Comment:
- [ ] Kindly make sure all the linters checkers are enabled, they help you applying best practice and prevent future bugs in your code. Kindly enable them